src/client.ts

```diff
@@ -128,6 +134,14 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     /* grab copies or valid objects */
     const defaultsOptions = options.defaults || {};
 
+    let allowedSessionTypes = options.allowedSessionTypes || [SessionTypes.softphone, SessionTypes.collaborateVideo, SessionTypes.acdScreenShare];
```

Comment:
NIT: this could probably be a `const`

Reply:
This can't be a const because it gets reassigned a few lines down if we are using jwt.